src/test/java/seedu/address/model/person/GoalTest.java

```diff
@@ -0,0 +1,107 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class GoalTest {
+
+    private static final List<String> VALID_WEEK_INPUTS = Arrays.stream(new String[]{
```

Comment:
`Arrays.asList(new String[] {"w", "week", "weekly"})` shorter line to convert to List